### ISSUE TITLE
[Mission Editor] Wrong descriptions for property type

### ISSUE THREAD

**Issue by Falki-git** [KSP2Community/editors#1]
![image](https://github.com/KSP2Community/editors/assets/72734856/218c0ce1-e742-478a-baec-fb06b1184049)

**Comment by Falki-git** [comment]
Possibly this as well?

![image](https://github.com/KSP2Community/editors/assets/72734856/36e6dea0-2c51-422b-81f5-0bf6eb9fbee7)

**Comment by jan-bures** [comment]
> ![image](https://private-user-images.githubusercontent.com/72734856/309581300-218c0ce1-e742-478a-baec-fb06b1184049.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk1MDA0NTcsIm5iZiI6MTcwOTUwMDE1NywicGF0aCI6Ii83MjczNDg1Ni8zMDk1ODEzMDAtMjE4YzBjZTEtZTc0Mi00NzhhLWJhZWMtZmIwNmIxMTg0MDQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzAzVDIxMDkxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyMjE0YjgxMjM0Mjk3OGViZjc1ZDUxODZkMzQzNzE1NWQ2NzUxZTk3NTI5YTA1NWRiYWY2MzE1Y2VhZmM3NDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.B3k-xIgVey_yfEy3re0UB3mrYZknFoSXMahbhHB6zmo)

Seems like the devs made a mistake here (since I extract these descriptions from the property watcher classes).
Fixed in 3b60260


> Possibly this as well?
> 
> ![image](https://private-user-images.githubusercontent.com/72734856/309581613-36e6dea0-2c51-422b-81f5-0bf6eb9fbee7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk1MDA0NTcsIm5iZiI6MTcwOTUwMDE1NywicGF0aCI6Ii83MjczNDg1Ni8zMDk1ODE2MTMtMzZlNmRlYTAtMmM1MS00MjJiLTgxZjUtMGJmNmViOWZiZWU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzAzVDIxMDkxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhYTU2MjhlM2NlMzA2ZjQ3NWUyOTk3ZGFmN2IwZmU4MjRlZmYwNGViYTdkZmZlMTAwNjk1Nzk5ZjYwODNlOTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cjaGIY7_dZQn3HsqY5FIyjYARKg8MZDhDQDYl5ncepQ)

This one actually has a correct description based on the code of VesselAudioSOIPropertyWatcher, they just named it very weirdly.

**Comment by Falki-git** [comment]
There also seems to be VesselSOIPropertyWatcher that does the same thing as VesselAudioPropertyWatcher currently

**Comment by jan-bures** [comment]
Yeah, this kinda seems like they just started writing VesselAudioPropertyWatcher by duplicating VesselSOIPropertyWatcher and forgot/didn't have time to finish it. I suppose I can just remove VesselAudioPropertyWatcher for now.
